docs/source/index.rst

```diff
@@ -44,6 +44,7 @@ please let us know at our :doc:`Support <Support>` page or by reaching out to us
     :glob:
 
     library/Library
+    EccFramework
```

Comment:
This is the wrong place to put the ECC library in the documentation. It should be under the User Guide section above.

Reply:
I moved it.